source/basic/module/src/named-import-alias.js

```diff
@@ -0,0 +1,3 @@
+// fooとして名前付きエクスポートされた変数をmyFooとしてインポートする
+import { foo as myFoo } from "./my-module.js";
```

Comment:
```suggestion
import { foo as myFoo } from "./named-export-alias.js";
```

セクション内で関係が成立してほしいので、こうしたほうがいいかも
<img width="848" alt="image" src="https://user-images.githubusercontent.com/19714/62171885-8114b000-b36b-11e9-94e3-4d73d0a36395.png">